ovs/lib/setup.py

```diff
@@ -779,6 +780,68 @@ def remove_node(node_ip, silent=None):
         SetupController._log(messages='Remove nodes finished', title=True)
 
     @staticmethod
+    def monitor_services():
+        """
+        Monitor the OVS services
+        :return: None
+        """
+        try:
+            previous_output = None
+            service_manager = check_output('cat /proc/1/comm', shell=True).strip()
```

Comment:
Why not offload this detection to the ServiceManager, then we have this detection code in only one place. Apart from that, I think it might be a better idea to move this code to the StorageRouterController as it "monitors" services running on a StorageRouter, and has not much to do with the Setup.